rust/rebase/src/subject/ethereum.rs

```diff
@@ -31,6 +31,10 @@ impl Subject for Eip155 {
         Ok(self.address.clone())
     }
 
+    fn verification_method(&self) -> Result<String, SubjectError> {
+        Ok(format!("{}#blockchainAddress", self.did()?))
```

Comment:
I was mistaken, it’s not blockchainAddress but blockchainAccountId (for the pkh:eth key-id). [here’s an example did doc](https://github.com/spruceid/ssi/blob/main/did-pkh/tests/did-eth.jsonld) for eth addresses. For Solana ones, it’s slightly different, [here’s an example of that](https://github.com/spruceid/ssi/blob/main/did-pkh/tests/did-sol.jsonld). not sure whether `controller` or `SolanaMethod2021` is better when getting the Solana verification method